### ISSUE TITLE
Request for a 4bit quantization model for omnilmm 12B

### ISSUE THREAD

**Issue by highkay** [OpenBMB/MiniCPM-V#53]
I have tried the multi card inference but failed, it seems the layers are not dispatched correctly.

```
        with init_empty_weights():
            model = OmniLMMForCausalLM.from_pretrained(model_name, tune_clip=True, torch_dtype=torch.bfloat16)
        model = load_checkpoint_and_dispatch(model, model_name, dtype=torch.bfloat16, 
                    device_map="balanced",  no_split_module_classes=['Eva','MistralDecoderLayer', 'ModuleList', 'Resampler']
        )
```

So it would be convenience to provide a quantization model for low memory cards, it may be nice if under 20GB single card or 40GB multi cards.

**Comment by jiayev** [comment]
Also want a quantization for omnilmm-12b. BNB seems not to work.

**Comment by iceflame89** [comment]
We released the more powerful [MiniCPM-Llama3-V 2.5](https://github.com/OpenBMB/MiniCPM-V/blob/main/README_en.md#minicpm-llama3-v-25) today，which has 8.5B parameters and much better performance， the int4 version is [here](https://github.com/OpenBMB/MiniCPM-V/blob/main/README_en.md#model-zoo). 
Welcome to use.
